core/data/src/main/java/kr/co/cotton/core/data/detailnews/repository/DefaultDetailNewsRepository.kt

```diff
@@ -0,0 +1,18 @@
+package kr.co.cotton.core.data.detailnews.repository
+
+import kotlinx.coroutines.Dispatchers
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between